package.json

```
@@ -28,7 +28,7 @@
     "jasmine": "^3.1.0",
     "jquery": "^3.4",
     "jsdom": "^15.2.1",
-    "node-sass": "^4.9.0",
+    "node-sass": "^4.12.0",
```

```suggestion
    "node-sass": "^4.13.1",
```

será posible usar la versión más nueva?

---

Si, ya la cambié